edit show list friends

minhieukt282 opened the pull request:
review

minhieukt282 committed:
edit show list friends

hale113 committed:
commit

hale113 committed:
Merge branch 'master' of https://github.com/minhieukt282/BE_Socail_media into hanhim

hale113 committed:
crud post

anhnbt commented on src/repo/relationshipRepo.ts:
```diff
@@ -0,0 +1,17 @@
+        })
+    }
+
+    findAll = async (id: string) => {
```
Thêm kiểu trả về cho method

anhnbt commented on src/service/user-service.ts:
```diff
@@ -1,10 +1,22 @@
 
 export class UserService {
     private randomId: RandomId
+    private relationshipService: RelationshipRepo
+    private accountService: AccountRepo
```
Tên biến sai ý nghĩa

anhnbt commented on src/service/user-service.ts:
```diff
@@ -1,10 +1,22 @@
+        this.accountService = new AccountRepo()
+    }
+
+    getFriends = async (username: string)=>{
```
1. Chưa format code
2. Thêm kiểu trả về cho method

anhnbt commented on src/service/user-service.ts:
```diff
@@ -1,10 +1,22 @@
+    getFriends = async (username: string)=>{
+        let account = await this.accountService.findByUsername(username)
+        let listFriends = await this.relationshipService.findAll(account[0].accountId)
+        console.log(listFriends)
```
Xóa console.log

anhnbt committed:
refactor code

- Add interface dto
- Thêm kiểu trả về cho method
- Định kiểu rõ ràng cho parameter
- Sửa lại status code chuẩn hơn

minhieukt282 committed:
Refactor code

refactor all code

minhieukt282 committed:
build: friend relationship

kết bạn, chấp nhận, từ chôí bạn bè

minhieukt282 committed:
built: friend relationship

thêm controller

minhieukt282 committed:
build: friend relationship

minhieukt282 committed:
add friend done

minhieukt282 committed:
Refactor code

minhieukt282 committed:
Refactor code

minhieukt282 committed:
before merge ha

minhieukt282 committed:
built socket io

minhieukt282 committed:
built logout

hale113 committed:
done crud post

minhieukt282 committed:
built like

minhieukt282 committed:
merge post
